runtime/development/src/lib.rs

```diff
@@ -889,10 +889,10 @@ parameter_types! {
 	pub const MaxSizeMetadata: u32 = 46; // length of IPFS hash
 
 	#[derive(scale_info::TypeInfo, Eq, PartialEq, Debug, Clone, Copy )]
-	pub const MaxTokenNameLength: u32 = 128;
+	pub const MaxTokenNameLength: u32 = MAX_TOKEN_NAME_LENGTH_BYTES;
 
 	#[derive(scale_info::TypeInfo, Eq, PartialEq, Debug, Clone, Copy )]
-	pub const MaxTokenSymbolLength: u32 = 128;
+	pub const MaxTokenSymbolLength: u32 = MAX_TOKEN_SYMBOL_LENGTH_BYTES;
```

Comment:
@mustermeiszer I would actually prefer these new constants to be `parameter_types` so that we can plug them directly into the `PoolSystem` instead of having to define `parameter_types` on each runtime and set them to these constants respectively.

What would be the best place to define those common consts?

Reply:
Probably, somewhere in libs/types crate. Maybe add a const module. WDYT?

Reply:
I was giving this a try but the current `MaxTokenSymbolLength` and `MaxTokenNameLength` is used and imported through the runtimes in a few places so I prefer to address this change in a different PR.